src/Symfony/Bridge/PhpUnit/Legacy/CoverageListenerTrait.php

```diff
@@ -78,13 +75,45 @@ public function startTest($test)
             }
         }
 
+        if (class_exists(Registry::class)) {
+            $this->addCoversForDocBlockInsideRegistry($test, $covers);
+
+            return;
+        }
+
+        $this->addCoversForClassToAnnotationCache($test, $covers);
+    }
+
+    private function addCoversForClassToAnnotationCache($test, $covers)
+    {
+        $testClass = \PHPUnit\Util\Test::class;
+        if (!class_exists($testClass, false)) {
+            $testClass = \PHPUnit\Util\Test::class;
+        }
```

Comment:
That doesn't make sense. You're setting `$testClass` to the same value in both cases.

Reply:
You're absolutely right. Getting this back.